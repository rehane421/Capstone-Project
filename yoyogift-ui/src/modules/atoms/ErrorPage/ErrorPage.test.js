import React from "react";
import { shallow } from "../../../enzyme";
import { shallowToJson } from "enzyme-to-json";
import { ErrorPage } from "./ErrorPage";

let props = {};
it("should render ErrorPage", () => {
  let wrapper = shallow(<ErrorPage {...props} />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
