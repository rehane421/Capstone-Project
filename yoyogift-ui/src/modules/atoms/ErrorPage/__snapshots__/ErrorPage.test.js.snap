// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ErrorPage 1`] = `
<img
  alt="404: Page not found"
  src="404.png"
  style={
    Object {
      "height": "100vh",
      "width": "100%",
    }
  }
/>
`;
